### ISSUE TITLE
Try to freeze DB-VAE layers

### ISSUE THREAD

**Issue by 88Mangos** [88Mangos/Polished-Capstone#1]

**Comment by 88Mangos** [comment]
That might require instantiation without custom training loop -> look if keras model compilers have a loss_fn argument to pass the debiasing loss function into
